daphne/Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde = { version = "^1.0", features = ["derive"] }
 rand = "0.8.5"
 thiserror = "1.0"
 url = { version = "2.2.2", features = ["serde"] }
+matchit = "0.6.0"
```

Comment:
"matchit" is only needed for tests, so it should go in "dev-dependencies" below. Otherwise it'll get built even if not needed.

Reply:
Thanks for pointing this out.